ui/guest/src/react/ExperienceBuilder.tsx

```diff
@@ -163,7 +166,16 @@ function ExperienceBuilderInternal(props: InternalGuestProps) {
   const [snack, setSnack] = useState<Partial<SnackbarProps>>();
   const dispatch = useDispatch();
   const state = useSelector<GuestState>((state) => state) as GuestState;
-  const { editMode, highlightMode, editModePadding, status, hostCheckedIn: hasHost, draggable, authoringBase } = state;
+  const {
+    editMode,
+    highlightMode,
+    editModePadding,
+    status,
+    hostCheckedIn: hasHost,
+    draggable,
+    authoringBase,
+    activeSite
+  } = state;
```

Comment:
I think this was a miss: `activeSite` is not in use.
```suggestion
const { editMode, highlightMode, editModePadding, status, hostCheckedIn: hasHost, draggable, authoringBase } = state;
```